duckduckgo/Android
# Major tracker networks summary

Asana Issue URL: https://app.asana.com/0/488551667048375/471718733961910

NOTE: There is an outstanding question around the maxcdn network covered in https://app.asana.com/0/488551667048375/494979307307237

## Description
Add major networks summary

## Steps to Test this PR:
1. Visit evanscycles.co.uk
1. Open the privacy dashboard (it will be a little slow as we haven't implemented live data yet and are just serializing to share data from one activity to the next)
1. Ensure that major tracker network count is displayed
